canonical/ubuntu.com
# feat: Build /20-04 page

## Done

- Build /20-04 page based off of [copydoc](https://docs.google.com/document/d/1lzxAL9Wt24m5-p5qG9y190rOg3FY5zVdKLYLlBoBcLU/edit?tab=t.0) and [figma design](https://www.figma.com/design/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=5107-21524&node-type=canvas&t=e97GwrAiLwPAWxc0-0)
- Added some features to `_latest_news_strip.html`
- Bumped vanilla to `4.18.4`

## QA

- Check out [demo link](https://ubuntu-com-14559.demos.haus/20-04)
- Check the design matches the [figma design](https://www.figma.com/design/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=5107-21524&node-type=canvas&t=e97GwrAiLwPAWxc0-0)
- Check the content matches the [copydoc](https://docs.google.com/document/d/1BSlOqwe--4MoVvAEok6WssBfEfwiRSXN1dNWCoUffdY/edit?tab=t.0#heading=h.mgfeqagpepfl)
- Check there is correct application of vanilla, specifically focusing on macro usage
- Check on different screen sizes
- Check the pages where `_latest_news_strip.html` is implemented and ensure there is no difference

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-17287
